specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -0,0 +1,2655 @@
+        "lastActionDateTime": null,
+        "message": null
+       }
+      }
+     },
+     "default": {
+      "description": "Error response.",
+      "schema": {
+       "$ref": "#/definitions/APIError"
+      }
+     }
+    },
+    "produces": [
+     "application/json"
+    ]
+   }
+  },
+  "/persongroups": {
+   "get": {
+    "description": "List person groups and their information.",
+    "operationId": "PersonGroups_List",
+    "parameters": [
+     {
+      "name": "start",
+      "in": "query",
+      "description": "List person groups from the least personGroupId greater than the \"start\". It contains no more than 64 characters. Default is empty. ",
+      "type": "string",
+      "enum": [
+       ""
+      ]
+     },
+     {
+      "name": "top",
+      "in": "query",
+      "description": "The number of person groups to list, ranging in [1, 1000]. Default is 1000. ",
+      "type": "integer",
+      "default": "1000",
+      "enum": [
+       "1000"
+      ]
+     },
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "A successful call returns an array of person groups and their information (personGroupId, name and userData).",
+      "schema": {
+       "$ref": "#/definitions/ListPersonGroupResponse"
+      },
+      "examples": {
+       "application/json": [
+        {
+         "personGroupId": "sample_group",
+         "name": "group1",
+         "userData": "User-provideddataattachedtothepersongroup"
+        },
+        {
+         "personGroupId": "sample_group2",
+         "name": "group2",
+         "userData": "User-provideddataattachedtothepersongroup"
+        }
+       ]
+      }
+     },
+     "default": {
+      "description": "Error response.",
+      "schema": {
+       "$ref": "#/definitions/APIError"
+      }
+     }
+    },
+    "produces": [
+     "application/json"
+    ]
+   }
+  },
+  "/persongroups/{personGroupId}/train": {
+   "post": {
+    "description": "Queue a person group training task, the training task may not be started immediately.",
+    "operationId": "PersonGroups_Train",
+    "parameters": [
+     {
+      "name": "personGroupId",
+      "in": "path",
+      "description": "Target person group to be trained.",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     }
+    ],
+    "responses": {
+     "202": {
+      "description": "A successful call returns an empty JSON body.  "
+     },
+     "default": {
+      "description": "Error response.",
+      "schema": {
+       "$ref": "#/definitions/APIError"
+      }
+     }
+    },
+    "produces": [
+     "application/json"
+    ]
+   }
+  },
+  "/facelists/{faceListId}": {
+   "put": {
+    "description": "Create an empty face list with user-specified faceListId, name and an optional userData. Up to 64 face lists are allowed to exist in one subscription.",
```

Comment:
`with user-specified faceListId, name and an optional userData` is unnecessary -- you can just describe what the method does and let the attributes and parameters speak for themselves.